frontend/components/UI/checkmark.tsx

```diff
@@ -1,33 +1,36 @@
 import { Box } from "@chakra-ui/react";
 import Check from "../../public/svg/check-mark.svg";
+import Cross from "../../public/svg/cross.svg";
+
 import { useState } from "react";
 
 interface AppProps {
     isColored: boolean;
 
     color: string;
     onClicking: () => void;
-    isUnfillable?: string;
+    isUncheckable?: string;
 }
 
-const CheckMark = ({ isColored, color, onClicking, isUnfillable }: AppProps) => {
-    const [isFilled, setIsFilled] = useState(isColored);
+const CheckMark = ({ isColored, color, onClicking, isUncheckable }: AppProps) => {
+    const [isChecked, setIsChecked] = useState(isColored);
 
     const clickHandler = () => {
-        if (isUnfillable === "true") {
+        if (isUncheckable === "true") {
             onClicking();
-            setIsFilled(!isFilled);
-        } else if (isUnfillable === "false" && isFilled === false) {
+            setIsChecked(!isChecked);
+        } else if (isUncheckable === "false" && isChecked === false) {
             onClicking();
-            setIsFilled(true);
+            setIsChecked(true);
         } else {
             return;
         }
     };
 
     return (
         <Box onClick={clickHandler}>
-            <Check fill={isFilled ? color : "black"} />
+            {isChecked && <Check fill={color} />}
+            {!isChecked && <Cross fill={"#E53945"} />}
```

Comment:
Just for a more intuitive UI, can we replace the cross with like a checkbox or something? And make the cursor become a pointer when it hovers over it so the user knows that it's clickable?